.github/workflows/config.json

```diff
@@ -44,12 +44,17 @@
         "fuzzy",
         "bert_all_MiniLM_L6_v2",
         "cosine",
-        "bert_distilbert_base_nli_stsb_mean_tokens",
-        "llm_answer_relevance",
-        "llm_context_precision"
```

Comment:
Hey @kcortinas Should these be removed? maybe Im missing something but is the goal to leverage ragas version of answer and context relevance as opposed to the existing implementation or have both versions to pick from? because it looks like `llm_answer_relevance` and `llm_context_relevance` is still being referenced in some scripts like `eval.py`

Reply:
Not exactly answering your question, but they're being removed from this particular config file (the config used for CI automation builds), but they can still be used by end users.

The sample config.json retains these items.

@kcortinas : is the idea to only test with these new metrics, and not with the older ones? Or, should we run CI tests with *all* metrics?

Reply:
ah got it.

Reply:
@prvenk This PR focuses on adding the RAGAS integration. Despite the name, some llm-based metrics are not the same as the corresponding RAGAS ones, another PR will follow to adjust and/or remove redundant metrics.

@martinpeck  Good point! Just kept the old metrics in the CI config but had to adjust the logging as we were hitting the max length for the “run_metrics” parameter (no need to log the “_run_metrics_” dict as we’re already logging the metrics). I haven’t added the metrics `"ragas_answer_relevancy"`, `"ragas_answer_correctness"` and `"ragas_answer_similarity"` yet as they require the deployment of the embedding model `“text-embedding-ada-002”` in the subscription we’re using to run the CI pipeline: @ritesh-modi  is this a model we can deploy? Thanks!

Reply:
I'm 99.9% sure it's already deployed, and being used for other things, so I think you can assume that we have this deployed already.

Reply:
I've also noticed that we were logging the `run_metrics` as dict and removed it in my [PR](https://github.com/microsoft/rag-experiment-accelerator/pull/529),
is that needed to be logged twice? @ritesh-modi

Reply:
I know customers might already use the old metrics,
quick question, but are those old metrics important to keep?
aren't ragas metrics enough?
keeping code means maintain it, along with unit tests and dependencies...
we should consider removing all of them if we are sure that RAGAS is enough.

Reply:
We can leave 2-3 metrics (ragas an non-rags) as an example in config.json. It need not be only ragas